core/src/test/python/target_configuration_helper_test.py

```diff
@@ -0,0 +1,56 @@
+"""
+Copyright (c) 2020, Oracle Corporation and/or its affiliates.  All rights reserved.
+The Universal Permissive License (UPL), Version 1.0
+"""
+
+import unittest
+
+import wlsdeploy.util.target_configuration_helper as HELPER
+import wlsdeploy.util.target_configuration as CONFIG
+
+from wlsdeploy.util.target_configuration import TargetConfiguration
+
+
+class TargetConfigurationTests(unittest.TestCase):
+    """
+
+    Test the target configuration and helper methods.
+    """
+
+    target_with_cred_name = None
+    target_without_cred_name = None
+
+    def setUp(self):
+        config = dict()
+        config[CONFIG.CREDENTIALS_METHOD] = 'secrets'
+        config[CONFIG.WLS_CREDENTIALS_NAME] = '__weblogic-credentials__'
+        self.target_with_cred_name = TargetConfiguration(config)
+
+        config = dict()
```

Comment:
duplicate with lines 24-27?

Reply:
two different configs.  changed the temporary variable names to avoid any sharing between classes during the test.